src/whisper_at_train/run_combined_training.sh

```
@@ -33,24 +46,48 @@ wa_start=36
+for file in "$tr_data" "$te_data" "$label_csv" "$pretrained_model"; do
+    if [ ! -f "$file" ]; then
+        echo "Error: Required file not found: $file"
+        exit 1
+    fi
+done
 
 # Get current timestamp in YYMMDDHHMM format
 timestamp=$(date +%y%m%d%H%M)
 
 exp_dir=./exp/combined-ft-${dataset}-${model}-${model_size}-${lr}-${lrscheduler_start}-${lrscheduler_decay}-ep${epoch}-bs${batch_size}-lda${lr_adapt}-ls${label_smooth}-mix${mixup}-${freqm}-${timem}-${timestamp}
 mkdir -p $exp_dir
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)